p2p/mock/MockMessageProcessor.go

```diff
@@ -0,0 +1,22 @@
+package mock
+
+import (
+	"fmt"
+
+	"github.com/ElrondNetwork/elrond-go/p2p"
+)
+
+type MockMessageProcessor struct {
+	Peer p2p.PeerID
+}
+
+func NewMockMessageProcessor(peer p2p.PeerID) MockMessageProcessor {
```

Comment:
normally New functions return pointer to object

Reply:
Fixed.